pset-functions.js

```diff
@@ -30,7 +30,28 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+function blackJack(playerCardScore, dealerCardScore) {
+  if (playerCardScore > dealerCardScore || (dealerCardScore > 21 && 21 - playerCardScore > 0)) {
```

Comment:
Natalie, your logic on this problem works perfectly! Great use of if/else statements to catch all potential scenarios. I also like that you went the extra mile and added a message as to who won the game! My only suggestion, is to try to condense what's in the conditional to make it a bit easier to read. An example of a similar/alternative solution can be found [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L34-L40).